force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -11813,6 +11837,54 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>pmtPaidPaymentDateRequired</shortDescription>
         <value>A paid Payment must have a Payment Date.</value>
     </labels>
+    <labels>
+        <fullName>pmtRefundPaymentErrorMessage</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>pmtPaidPaymentDateRequired</shortDescription>
+        <value>There was an error processing the refund.</value>
+    </labels>
+    <labels>
+        <fullName>pmtPaymentNotFund</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Cannot find the payment in Elevate</shortDescription>
+        <value>Can not find the corressponding payment record in Elevate.</value>
+    </labels>
+    <labels>
+        <fullName>pmtPaymentNotRefundable</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Cannot refund the payment</shortDescription>
+        <value>Cannot refund a transaction twice or refund a refund.</value>
+    </labels>
+    <labels>
+        <fullName>pmtRefundPaymentMessage</fullName>
+        <categories>Payment</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Refund Information</shortDescription>
+        <value>This will refund the payment made on {{VALUE}} in the amount of {{VALUE}}</value>
```

Comment:
Can we follow the existing convention to use {0} and {1} here? Also, if this label get split up, it may become difficult to translate.

Reply:
that's a really good point on the translation part. Another way is to use `String.replace()` with hardcoded the amount and date. However, we will lose the ability to automatically convert Date and Currency. I will connect with Grace when she is back to see if we can make the UI change to separate the sentence.